### DISCUSSION TITLE
Question re Java Runtime 17

### DISCUSSION THREAD

**Discussion by wordmuse** [freeplane/freeplane#309] [Help] [answered]
I just tried to install Freeplane 1.10 and ran into a problem. 

Freeplane 1.10 installed just fine, but when I ran it, I got an error: Needs Java Runtime 17. 

I clicked OK and was taken to Java's download page, here: https://java.com/en/download/

I downloaded and successfully installed the latest version of Java. I tried running FP1.10 again and got the same error message.

So I'm back to using FP1.19.14, which is fine for now.

Thoughts?

~ Bal

**Comment by rexelbartolome** [comment, marked as answer]
I think there's a problem with the environment variable etc. but to save you the headache, Dimitry already provided an option that bundles the needed Java stuff in the installation :D So just pick this one or anything that says "with Java"

![image](https://user-images.githubusercontent.com/88552647/162871406-ca8fc0b1-1036-40c9-a14b-d770179f4ea0.png)

**Reply by wordmuse** [reply]
Thank you very much rexelbartolome - and thank you Dimitry!  I've now got FP1.10.1 running just fine.

Best Regards,
~ Bal

**Comment by ab1kenobee** [comment]
Hello Members:

I have (2) questions regarding FREEPLANE:

1. Will it run on Zorin 16.2 OS Linux (with Java)?
2. Can it import or in any way use mind maps from The Brain (http://www.thebrain.com)?

Thank you, Alan.

**Reply by euu2021** [reply]
> Can it import or in any way use mind maps from The Brain (http://www.thebrain.com)?

@wordmuse , I think you know The Brain well, right? How would you export The Brain mind maps into FreePlane?

**Comment by wordmuse** [comment]
Happy New Year!

TB13 has an option to import FreeMIND files. Both Freepland and Freemind
use files with a .mm extension. But the file structures must be different
because I just tried importing a Freeplane file and got an error message.
Freeplane can export to a text file and TB13 can import a text file. But
you'd have to play with it to see if it gives you what you want.

Sorry I can't be more helpful.

~ Bal

**Reply by ab1kenobee** [reply]
Thank you.
